### ISSUE TITLE
Impossible to get true Center Crop 

### ISSUE THREAD

**Issue by bonlime** [NVIDIA/DALI#1111]
Somehow Related to #523 
When using any DALI Crop operator there is only an option to pass `crop_pos_x` and `crop_pos_y` using which it is impossible to get a true Center Crop which is required for measuring performance on Imagenet. I've used proposed pipeline from the docs/resnet_example and measured performance of the pretrained models in pytorch and keras and it was persistenly 1-1.5% worse that stated due to the wrong croping.

**Comment by awolant** [comment]
Hi, thanks for the question. Could you please tell which exactly pipeline you used? I'm assuming the one from PyTorch example?
Also with what reference code are you comparing it?
I will look through our CI for possible accuracy regressions.

**Comment by bonlime** [comment]
Yes, I compared it with official Imagenet example. There they use: `transforms.Resize(256), transforms.CenterCrop(224)` for validation augmentations which is a standard way of validation on Imagenet

**Comment by bonlime** [comment]
I just realized I was wrong and passing `crop_pos_x=0.5, crop_pos_y=0.5` gives me a true center crop. the difference appears to be in the interpolation method used. Passing `dali.types.INTERP_TRIANGULAR` to resize gives +1-1.5% and makes result nearly identical

**Comment by awolant** [comment]
Great, I'm glad it worked for you.

**Comment by makslevental** [comment]
@bonlime why not use `INTERP_CUBIC`? @awolant correct me if i'm wrong but that is bicubic interpolation?

**Comment by bonlime** [comment]
@makslevental
I used pretrained weight, which were trained with `BILINEAR` interpolation. Using `BICUBUC` requires retraining, so I decided to stick to `BILINEAR`

**Comment by makslevental** [comment]
@bonlime ah i understand.
